### DISCUSSION TITLE
How do I change where the wasm app listens to for example my IP: 192.168.0.10 Thanks

### DISCUSSION THREAD

**Discussion by ImparTechnologyCeo** [AvaloniaUI/Avalonia#11933] [General]
App url: http://localhost:5000/index.html
App url: https://localhost:5001/index.html

**Comment by maxkatz6** [comment]
You can change it in the launchSettings.json file if you need it for local testing. This file is created/managed by .NET SDK, so they should have documentation on it.

**Reply by ImparTechnologyCeo** [reply]
Thank you very much, I had not seen this file that is inside Properties, I had been working a little toasted for days and honestly I knew that but I could not find the file. Thanks
